config_gen.py

```diff
@@ -202,6 +202,25 @@ def run(cmd, *args, **kwargs):
         print("$ " + " ".join(cmd))
         subprocess.call(cmd, *args, **kwargs)
 
+    # helper function to set CC and CXX variables correctly on OSX with qmake
+    def patch_qmake_makefile(makefile_path):
+        compilers = {
+            "CC": "clang",
+            "CXX": "clang++",
+            "LINK": "clang++"
+        }
+        patched_makefile_lines = []
+        with open(makefile_path, "r") as f:
```

Comment:
On the other hand, the patch is applied to a temporary file, so actually style does not matter here. I will change the code.